### ISSUE TITLE
(Windows 10/Firefox) Cards with links are read as "Link Box"

### ISSUE THREAD

**Issue by mdctenga** [grommet/hpe-digitaltoolkit#160]
Issue also affects Chrome and Internet Explorer on Windows 10.

Using [NVDA screen reader on PC](http://www.nvaccess.org/), when the user uses their arrow keys to navigate over cards with links, instead of reading the content of the card, the screen reader reads, "link box".

If card does not have a link, NVDA reads the content.

**Comment by davidcalhoun** [comment]
I'm not 100% sure about this, but after pressing the down arrow and hearing "link box", if the user is interested in getting info about the link, then they press tab for "next link".

Here's how I got it to read out the contents:

Down Arrow -> "link box" -> Tab -> "circle play watch now link"

**Comment by davidcalhoun** [comment]
I misread the original issue.  Here's a simplified Codepen illustrating the issue: http://codepen.io/franksvalli/pen/mAgjmG?editors=0010

Looks like removing aria-label fixes the issue
